src/Semantics.js

```diff
@@ -369,8 +370,9 @@ function newDefaultAction(type, name, doIt) {
     } else {
       // Otherwise, we throw an exception to let the programmer know that we don't know what
       // to do with this node.
-      throw new Error(
-          'Missing semantic action for ' + this.ctorName + ' in ' + name + ' ' + type);
+
+      throw errors.semanticErrors(
+          this.ctorName, name, type) ;
```

Comment:
Yes, good question. I was think we could just throw the error only for the top-level wrapper, but after looking at it, it's trickier than I thought.

Let me look at this a bit more and get back to you.